website/newsletters/emails.py

```diff
@@ -38,32 +33,23 @@ def send_newsletter(newsletter):
     main_partner = Partner.objects.filter(is_main_partner=True).first()
     local_partners = services.split_local_partners()
 
-    with mail.get_connection() as connection:
-        language = ("en", "English")
-        translation.activate(language[0])
-
-        context = {
-            "newsletter": newsletter,
-            "agenda_events": events,
-            "main_partner": main_partner,
-            "local_partners": local_partners,
-            "lang_code": language[0],
-        }
-        context.update(thumbnail_sizes(None))
-
-        try:
-            send_email(
-                to=[f"newsletter@{settings.GSUITE_DOMAIN}"],
-                subject=newsletter.title,
-                txt_template="newsletters/email.txt",
-                html_template="newsletters/email.html",
-                from_email=settings.NEWSLETTER_FROM_ADDRESS,
-                connection=connection,
-                context=context,
-            )
-
-            logger.info("Sent %s newsletter", language[1])
-        except SMTPException:
-            logger.exception("Failed to send the %s newsletter", language[1])
+    context = {
+        "newsletter": newsletter,
+        "agenda_events": events,
+        "main_partner": main_partner,
+        "local_partners": local_partners,
+    }
+
+    try:
+        send_email(
```

Comment:
I think this won't reuse the smtp connection, so it will be very slow. Maybe we should update `send_email` with some caching on the smtp connection

Reply:
We could do some global long term connection caching but if that's useful I reckon django would support that better